conda/exceptions.py

```diff
@@ -1237,7 +1238,11 @@ def _execute_upload(self, error_report):
             'User-Agent': self.user_agent,
         }
         _timeout = self.http_timeout
+        username = getpass.getuser()
+        error_report['is_unicode'] = True if isinstance(username, bytes) else False
```

Comment:
I think a better way to express this might be "does the username contain characters that are outside of the ascii set?"

https://stackoverflow.com/questions/196345/how-to-check-if-a-string-in-python-is-in-ascii